blog/_posts/2025-07-02-kaito-distributed-inference.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "From 7B to 70B+: Serving giant LLMs efficiently with KAITO and ACStor v2"
+date: 2025-07-02
+authors:
+- Sachi Desai
+- Francis Yu
+categories: 
+- ai
+- storage
+---
+
+XL-size large language models (LLMs) are quickly evolving from experimental tools to essential infrastructure. Their flexibility, ease of integration, and growing range of capabilities are positioning them as core components of modern software systems. 
+
+Massive LLMs power virtual assistants and recommendations across social media, UI/UX design tooling and self-learning platforms. But how do they differ from your average language model? And how do you get the best bang for your buck running them at scale?
+
+Let’s unpack why large models matter how pairing them with Kubernetes and NVMe local storage accelerates your intelligent app development.
+
+# Aren’t LLMs large enough?
+
+Large models aren’t just big for show — they’re smart, efficient, and versatile thanks to:
+
+- Efficient transformer architectures powering them
+- Compatibility with high-performance inference libraries like vLLM
+- Better scores on tasks like instructions, coding, math, multilingual understanding, and long-context memory
+
+Running these XL-size LLMs at scale can be more cost-effective than relying on commercial APIs, if you play your cards right.
+
+# When self-hosting big models makes sense 💡
+
+Self-hosting LLMs on Kubernetes is growing in popularity for organizations that are:
+
+- Running lots of inference: batch jobs, chatbots, agents, or apps
+- Have access to commercial GPUs like NVIDIA H100 or A100
+- Want to avoid per-token API fees (which easily skyrocket costs at scale)
+- Keen to fine-tune or customize the model — something closed APIs usually block
+- Have sensitive or proprietary data to keep ringfenced and protected from accidental exposure through third-party logs
+
+Self-hosting with the Kubernetes AI Toolchain Operator (KAITO) helps you achieve all this and more! KAITO is a CNCF Sandbox project that simplifies and optimizes your inference and tuning workloads on Kubernetes. It integrates with vLLM, a high-throughput LLM inference engine optimized for serving large models efficiently. 
+
+vLLM supports quantized models (like 4-bit GPTQ or AWQ), reducing memory/GPU requirements drastically without major accuracy trade-offs. KAITO’s modular, plug-and-play setup allows you to go from model selection to production-grade API quickly:

```

Comment:
What's the expectation that the reader would know what those are? Perhaps a link?